### PR TITLE
Generate new README

### PR DESCRIPTION
**Pull request by Bence1001**
### Checklist

- [x] I've read and accepted the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- Requires _PATCH_ [version update](https://semver.org/)

### Context
* We are introducing a new README format for steps.

### Changes
* Added `generate-readme` workflow.
* Generated new README.

### CODE REVIEW

**Review by mateherber — APPROVED** [review]

File: bitrise.yml [hunk 1]
```diff
@@ -4,32 +4,36 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
...
+        - branch: $TEST_APP_BRANCH
+        - clone_into_dir: ./_tmp
+    - path::./:
+        title: Step Test
+        inputs:
+        - project_path: ./_tmp/$BITRISE_PROJECT_PATH
+
+  generate_readme:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-readme-generator.git@main:
```
Comment (mateherber) [inline bitrise.yml:39]: Do we have to write `main` if that is the "default" branch?

Comment (Bence1001) [reply bitrise.yml:39]: Sadly, yes: https://bitfall.slack.com/archives/CT108S18S/p1629716232037200?thread_ts=1629715585.035900&cid=CT108S18S.

Comment (Bence1001) [reply bitrise.yml:39]: Sadly yes. I'll send you the link to the internal discussion on Slack.

**tl;dr:** Bitrise only supports `master` as the default branch during `bitrise.yml` parse at the moment.
